servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSResources.java

```diff
@@ -17,12 +17,15 @@
     @Path("/domain/{domain}")
     @Produces(MediaType.APPLICATION_JSON)
     public Domain getDomain(@PathParam("domain") String domain) {
+        int code = 200;
+        ResourceContext context = this.delegate.newResourceContext(this.request, this.response);
+        Object timerMetric = this.delegate.startMetricTiming();
         try {
-            ResourceContext context = this.delegate.newResourceContext(this.request, this.response);
             context.authenticate();
-            return this.delegate.getDomain(context, domain);
+            Domain returnedValue = this.delegate.getDomain(context, domain);
```

Comment:
not sure why this change was made?

Reply:
Mistake. I originally planned on stopping the timer just before returning the value (before figuring I can stop it in finally and have the error code if needed)